.clang-format

```diff
@@ -1,23 +1,18 @@
+---
+# We'll use defaults from the LLVM style, but with the following differences.
 BasedOnStyle: LLVM
-ColumnLimit: 100
-UseTab: Never
-TabWidth: 2
-IndentWidth: 2
-IncludeBlocks: Preserve
-NamespaceIndentation: None
-PointerAlignment: Left
 AlignAfterOpenBracket: AlwaysBreak
 AlignEscapedNewlines: Left
+AlignOperands: false
+AlignTrailingComments: false
 AllowAllParametersOfDeclarationOnNextLine: false
-PenaltyReturnTypeOnItsOwnLine: 300
 BinPackArguments: false
 BinPackParameters: false
-IndentCaseLabels: true
-BreakBeforeTernaryOperators: true
 BreakBeforeBinaryOperators: All
 BreakBeforeBraces: Allman
+ColumnLimit: 100
 Cpp11BracedListStyle: false
-AlignConsecutiveAssignments: false
```

Comment:
You removed these two.  Can you elaborate on why

Reply:
These values are identical to the LLVM defaults. The only values we need to specify are those where we differ.

Reply:
Good question. Keen eye :)